stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -358,6 +363,11 @@ public class MessageListView : ConstraintLayout {
 
     private val DEFAULT_ATTACHMENT_CLICK_LISTENER =
         AttachmentClickListener { message, attachment ->
+            val hasInvalidAttachments = message.attachments.any { it.uploadState != null }
```

Comment:
This feels much better to me.

But we still have the issue of: do we block opening until _all_ attachments are uploaded vs blocking only the uploaded attachments. 

This could mess up the preview too if there are images, so I propose we just don't allow users to open anything if all the attachments aren't uploaded.